### PR TITLE
saaa

### PR DESCRIPTION
**Pull request by ppezaris**
fff

### CODE REVIEW

**Comment by marcelofarias** [comment]
comment from intellij

File: shared/agent/src/providers/registry.ts [hunk 1]
```diff
@@ -379,26 +379,28 @@ export class ThirdPartyProviderRegistry {
 		return response;
 	}
 
-	@log()
+	@log({
+		prefix: (context, args) => `${context.prefix}:${args.method}`
+	})
```
Comment (marcelofarias) [inline shared/agent/src/providers/registry.ts:384]: single comment from intellij

Comment (marcelofarias) [reply shared/agent/src/providers/registry.ts:384]: reply from intellij

File: shared/agent/src/managers/postsManager.ts [hunk 2]
```diff
@@ -883,14 +888,152 @@ export class PostsManager extends EntityManagerBase<CSPost> {
...
+	): Promise<CreateShareableCodemarkResponse | CreatePassthroughCodemarkResponse | undefined> {
 		const codemarkRequest: CreateCodemarkRequest = {
 			...request.attributes,
 			status:
 				request.attributes.type === CodemarkType.Issue ? request.attributes.status : undefined,
 			markers: []
 		};
 
+		if (
+			request.textDocuments &&
```
Comment (marcelofarias) [inline shared/agent/src/managers/postsManager.ts:900]: comment in review from intellij

File: shared/agent/src/managers/postsManager.ts [hunk 3]
```diff
@@ -82,6 +85,8 @@ import { EntityCache, EntityCacheCfg } from "./cache/entityCache";
 import { EntityManagerBase, Id } from "./entityManager";
 import { MarkerLocationManager } from "./markerLocationManager";
 import { MarkerCreationDescriptor, MarkersBuilder } from "./markersBuilder";
+import * as csUri from "../system/uri";
```
Comment (marcelofarias) [inline shared/agent/src/managers/postsManager.ts:88]: another single comment from intellij

File: atom/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "codestream",
 	"main": "./dist/codestream",
-	"version": "8.2.1",
+	"version": "8.3.4",
 	"description": "Discuss code easily. Streamline Code Reviews. Improve code quality. Works w/ Slack, Teams, Jira, Trello, git, and more.",
```
Comment (bcanzanella) [inline atom/package.json:5]: does it update?

File: jb/bin/jb-publish-jcef [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+	tagRepo    "--no-tag"     0  1
+parse_options "$@"
+eval set -- ${PARGS_POSITIONAL[@]}
+
+infoFileCount=`ls $JB_TOP/codestream-jb-jcef-*.info|wc -l`
+[ $infoFileCount -ne 1 ] && echo "$infoFileCount info files found" >&2 && exit 1
+infoFile=`echo $JB_TOP/codestream-jb-jcef-*.info`
+echo "infoFile=$infoFile"
+
+assetFileCount=`ls $JB_TOP/codestream-jb-jcef-*.zip|wc -l`
```
Comment (bcanzanella) [inline jb/bin/jb-publish-jcef:16]: does it really update?

File: jb/docs/change-notes.html [hunk 6]
```diff
@@ -1,3 +1,49 @@
...
+</ul>
+<h5>Fixed</h5>
+<ul>
+    <li>Fixes an issue with certain scenarios not suggesting appropriate default reviewers using the "authorship" model</li>
+</ul>
+<h4 id="8_3_3">8.3.3</h4>
+<h5>Fixed</h5>
+<ul>
+    <li>Addresses <a href="https://github.com/TeamCodeStream/CodeStream/issues/234">#234</a> - Bug with Code range function</li>
+    <li>Fixes an issue with the handling of invalid Jira queries</li>
```
Comment (bcanzanella) [inline jb/docs/change-notes.html:14]: foo

File: jb/src/main/kotlin/protocols/webview/HostProtocol.kt [hunk 7]
```diff
@@ -147,3 +148,22 @@ class UpdateServerUrlRequest(
 class OpenUrlRequest(
     val url: String
 )
+
+class CompareLocalFilesRequest(
+    val repoId: String,
+    val filePath: String,
+    val headSha: String,
+    val headBranch: String,
```
Comment (bcanzanella) [inline jb/src/main/kotlin/protocols/webview/HostProtocol.kt:156]: bar

Comment (bcanzanella) [inline jb/src/main/kotlin/protocols/webview/HostProtocol.kt:156]: 111

File: jb/src/main/kotlin/protocols/webview/HostProtocol.kt [hunk 8]
```diff
@@ -147,3 +148,22 @@ class UpdateServerUrlRequest(
...
+    val baseSha: String,
```
Comment (bcanzanella) [inline jb/src/main/kotlin/protocols/webview/HostProtocol.kt:157]: 222

File: jb/src/main/kotlin/review/ReviewService.kt [hunk 9]
```diff
@@ -1,13 +1,13 @@
 package com.codestream.review
```
Comment (bcanzanella) [inline jb/src/main/kotlin/review/ReviewService.kt:1]: cheese

File: jb/src/main/kotlin/review/ReviewService.kt [hunk 10]
```diff
@@ -1,13 +1,13 @@
...
 
 import com.codestream.agentService
-import com.codestream.protocols.agent.GetAllReviewContentsParams
+import com.codestream.protocols.agent.GetFileContentsAtRevisionParams
 import com.codestream.protocols.agent.GetLocalReviewContentsParams
 import com.codestream.protocols.agent.GetReviewContentsResult
+import com.codestream.protocols.webview.Context
```
Comment (bcanzanella) [inline jb/src/main/kotlin/review/ReviewService.kt:7]: aaa

File: jb/.idea/prettier.xml [hunk 11]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PrettierConfiguration">
+    <option name="myRunOnSave" value="true" />
```
Comment (bcanzanella) [inline jb/.idea/prettier.xml:4]: q1

File: jb/bin/jb-publish-jcef [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+# special handling of jcef asset - tag and publish
+. $DT_TOP/lib/parse_opts.sh
+
+parse_defaults \
+	tagRepo    "--no-tag"     0  1
+parse_options "$@"
+eval set -- ${PARGS_POSITIONAL[@]}
+
+infoFileCount=`ls $JB_TOP/codestream-jb-jcef-*.info|wc -l`
+[ $infoFileCount -ne 1 ] && echo "$infoFileCount info files found" >&2 && exit 1
```
Comment (bcanzanella) [inline jb/bin/jb-publish-jcef:12]: pls work?

Comment (bcanzanella) [reply jb/bin/jb-publish-jcef:12]: test

File: jb/.idea/prettier.xml [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+  </component>
```
Comment (bcanzanella) [inline jb/.idea/prettier.xml:5]: aaa

**Comment by bcanzanella** [comment]
qqq

File: jb/build.gradle [hunk 14]
```diff
@@ -6,14 +6,14 @@ plugins {
 }
 
 group "com.codestream"
-version "8.2.1"
+version "8.3.5"
```
Comment (bcanzanella) [inline jb/build.gradle:9]: hi pez

File: jb/docs/change-notes.html (hunk 6 above)
Comment (bcanzanella) [inline jb/docs/change-notes.html:14]: aaa

File: jb/src/main/kotlin/agent/AgentService.kt [hunk 15]
```diff
@@ -11,6 +11,8 @@ import com.codestream.protocols.agent.DocumentMarkersParams
 import com.codestream.protocols.agent.DocumentMarkersResult
 import com.codestream.protocols.agent.GetAllReviewContentsParams
 import com.codestream.protocols.agent.GetAllReviewContentsResult
+import com.codestream.protocols.agent.GetFileContentsAtRevisionParams
```
Comment (bcanzanella) [inline jb/src/main/kotlin/agent/AgentService.kt:14]: a

File: shared/agent/package-lock.json [hunk 16]
```diff
@@ -1186,7 +1186,7 @@
                 },
                 "util": {
                     "version": "0.10.3",
-                    "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
+                    "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
```
Comment (bcanzanella) [inline shared/agent/package-lock.json:1189]: 1189

File: atom/CHANGELOG.md [hunk 17]
```diff
@@ -1,5 +1,45 @@
 # Change Log
 
+## [8.3.3] - 2020-8-21
+
```
Comment (bcanzanella) [inline atom/CHANGELOG.md:4]: test

File: atom/CHANGELOG.md [hunk 18]
```diff
@@ -1,5 +1,45 @@
...
+### Fixed
+
+- Fixes an issue with certain scenarios not suggesting appropriate default reviewers using the "authorship" model
+
+## [8.3.2] - 2020-8-13
+
+### Fixed
```
Comment (ppezaris) [inline atom/CHANGELOG.md:11]: jjust commenting

**Pull request closed without merging**